### PR TITLE
[AC][SUPP-667-668-669-670-671-624] email+submit VAT

### PR DESCRIPTION
**Pull request by AndyHWChung**

### CODE REVIEW

File: test/controllers/DetermineEligibilityControllerSpec.scala [hunk 1]
```diff
@@ -34,64 +35,92 @@ class DetermineEligibilityControllerSpec extends ItSpec {
 
   "Determine eligibility endpoint should route user correctly and send an audit event" - {
     forAll(Table(
-      ("Scenario flavour", "eligibility rules", "ineligibility reason audit string", "expected redirect", "updated journey json"),
+      ("Scenario flavour", "eligibility rules", "ineligibility reason audit string", "expected redirect", "updated journey json", "tax regime"),
```
Comment (barnesjake) [inline test/controllers/DetermineEligibilityControllerSpec.scala:38]: tiny thing, but the new field isn't taxRegime, but origin instead

Comment (AndyHWChung) [reply test/controllers/DetermineEligibilityControllerSpec.scala:38]: oh yeah, good spot - I"ll change it

File: test/testsupport/testdata/TdAll.scala [hunk 2]
```diff
@@ -268,66 +312,76 @@ object TdAll {
...
     Some(List(CustomerDetail(Some(Email(SensitiveString(email))), Some(source))))
 
   val someRegimeDigitalCorrespondenceFalse: Option[RegimeDigitalCorrespondence] = Some(RegimeDigitalCorrespondence(false))
   val someRegimeDigitalCorrespondenceTrue: Option[RegimeDigitalCorrespondence] = Some(RegimeDigitalCorrespondence(true))
 
-  val arrangementResponse: ArrangementResponse = ArrangementResponse(ProcessingDateTime("2022-03-23T13:49:51.141Z"), CustomerReference("123PA44545546"))
+  def arrangementResponse(taxRegime: TaxRegime): ArrangementResponse = {
+    val customerReference = taxRegime match {
+      case TaxRegime.Epaye => CustomerReference("123PA44545546")
+      case TaxRegime.Vat   => CustomerReference("123PA44545546")
```
Comment (barnesjake) [inline test/testsupport/testdata/TdAll.scala:383]: Should the VAT customer ref be `101747001` just for consistency (seen that one used below)? I don't mind though

**Review by barnesjake — DISMISSED** [review]

File: test/testsupport/testdata/TdAll.scala (hunk 2 above)
Comment (AndyHWChung) [reply test/testsupport/testdata/TdAll.scala]: fixed [here](https://github.com/hmrc/essttp-frontend/pull/264/commits/c7d07634174ecb39cffd22bfc025cd0fbab42c7e)

**Review by barnesjake — APPROVED** [review]

**Pull request merged**